README.md

```diff
@@ -1,73 +1,83 @@
-   and check constraints
- * MsgPack data format and MsgPack based
-   client-server protocol
- * two data engines: 100% in-memory with
-   optional persistence and an own implementation of LSM-tree, 
-   to use with large data sets
- * multiple index types: HASH, TREE, RTREE, BITSET
- * asynchronous master-master replication
- * authentication and access control
- * the database is just a C extension to the
-   application server and can be turned off
-
-Supported platforms are Linux/x86, FreeBSD/x86 and OpenBSD/x86, Mac OS X.
-
-Tarantool is ideal for data-enriched components of
-scalable Web architecture: queue servers, caches,
-stateful Web applications.
-
-To download and install Tarantool as a binary package for your OS, please visit
-https://tarantool.io/en/download/.
-
-To build Tarantool from source, see detailed instructions in the Tarantool
-documentation at https://tarantool.io/en/doc/2.1/dev_guide/building_from_source/.
-
-Please report bugs at https://github.com/tarantool/tarantool/issues
-We also warmly welcome your feedback in the discussion mailing
-list, tarantool@googlegroups.com.
+
+* MsgPack data format and MsgPack based client-server protocol
+* two data engines: 100% in-memory with optional persistence and an own
+  implementation of LSM-tree, to use with large data sets
+* multiple index types: HASH, TREE, RTREE, BITSET
+* document oriented JSON path indexes
+* asynchronous master-master replication
+* synchronous quorum-based replication
+* RAFT-based automatic leader election
+* authentication and access control
+* ANSI SQL, including views, joins, referential and check constraints
+* [connectors][connectors] for many programming languages
+* the database is just a C extension to the application server and can be
+  turned off
+
+Supported platforms are Linux (x86_64, aarch64), Mac OS X (x86_64, M1), FreeBSD
+(x86_64).
+
+Tarantool is ideal for data-enriched components of scalable Web architecture:
+queue servers, caches, stateful Web applications.
+
+To download and install Tarantool as a binary package for your OS or using
+docker, please visit [download instructions][download].
+
+To build Tarantool from source, see detailed [instructions][building] in the
+Tarantool documentation.
+
+Please report bugs at [issue tracker][issue-tracker]. We also warmly welcome
+your feedback on the [discussions][discussions-url] page and questions on
+[Stack Overflow][stackoverflow-url].
+
+We accept contributions via pull requests. Check out our [How to get
+involved][get-involved] guide.
 
 Thank you for your interest in Tarantool!
 
-[travis-badge]: https://api.travis-ci.org/tarantool/tarantool.svg?branch=master
-[travis-url]: https://travis-ci.org/tarantool/tarantool
-[gitlab-ci-badge]: https://gitlab.com/tarantool/tarantool/badges/master/pipeline.svg
-[gitlab-ci-url]: https://gitlab.com/tarantool/tarantool/commits/master
+----
+
+Also, don't miss our [Awesome Tarantool][awesome-list] list of delightful
```

Comment:
I see no reason to separate this sentence from the rest of the text. Do you think it is less important than the references above?

Reply:
I didn't know where it is better to inject it. We had the following document flow:

* features;
* platforms;
* use cases;
* download;
* build;
* report, discuss, ask;
* contribute.

I have added the awesome list link after the 'build' paragraph and rephrased it to the style 'to <..do something..>, <..do this..>'.